bin/pac

```diff
@@ -24,6 +24,8 @@ Usage: pac [options] <command> <args>...
   pac remove <package>
   pac update [args]...
   pac upgrade [args]...
+  pac list-installed				List locally installed packages
+  pac locate FILE				Query the package which provides FILE          
```

Comment:
Could you change `FILE` to `<file>`, so it's consistent with the other command documentation?